src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Marshallers/ExceptionMarshaller.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.InteropServices.JavaScript.Private
+{
+
+    /// <summary>
+    /// Always creating stack trace is more expensive than passing js_handle and GC of JSObject.
+    /// Same in reverse direcion with gc_handle and ManagedObject.
+    /// The Exception/JSException instances roundtip and keep identity.
+    /// </summary>
+    internal unsafe class ExceptionMarshaller : BaseMarshaller<Exception>
+    {
+#nullable disable
+        public override bool UseRoot => true;
+        public override MarshalToManagedDelegate ToManaged => (ref JavaScriptMarshallerArg arg) => JavaScriptMarshalImpl.MarshalToManaged(ref arg, default(Exception));
```

Comment:
this nested ```=> (...) =>``` syntax is really confusing especially on one line, maybe do a line break after the first =>

Reply:
Changed to properties for better readability.